client/js/widgets/widget.js

```diff
@@ -93,17 +95,25 @@ export class Widget extends StateManaged {
     this.domElement.addEventListener('mouseleave',  e => this.hideEnlarged(e), false);
     this.domElement.addEventListener("touchstart", e => this.touchstart(), false);
     this.domElement.addEventListener("touchend", e => this.touchend(), false);
+    this.domElement.addEventListener('mousedown', e => this.selected(), false);
+    this.domElement.addEventListener('mouseup', e => this.selectedEnd(), false);
 
     this.touchstart = function() {
       if (!this.timer) {
         this.timer = setTimeout(this.onlongtouch.bind(this), 500, false);
       }
+      if (!this.addOverlay || window.getComputedStyle(addOverlay).display === 'none')
+        this.set('selectedBy', playerName);
```

Comment:
Why don't you just call `this.selected()` here (and `this.selectedEnd()` below)?